app/assets/stylesheets/restaurants.scss

```diff
@@ -1,3 +1,112 @@
-// Place all the styles related to the Restaurants controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin make-float-boxes(){
+  float: left;
+  display: inline-block;
+}
+
+@mixin image-no-repeat($url){
+  background-image: url($url);
+  background-repeat: no-repeat;
+
+}
+.header{
+  position: relative;
+  width: 100%;
+  height: 200px;
+  background-size: cover;
+  @include image-no-repeat('header.png');
+
+  .header-text {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    top: 25%;
+    background-position: center;
+    @include image-no-repeat('header-text.png');
+  }
+
+}
+
+.list{
```

Comment:
Nitpicking: make sure to space before the braces `.class {` instead of `.class{`